::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Thu, 7 Jun 2018 21:19:07 +0200
Subject: [PATCH] linux.system.repo: Drop arch for mcp.mirror repo

Extending the arch list to 'amd64,arm64' would lead to doubling repo
metadata downloads on all archs, while dropping it will allow APT to
only download the repo metadata for the current system arch.

Setting the new linux_system_architectures is also problematic in the
context of mixed-architecture clusters.

To keep old behavior for amd64 and prevent multiple downloads, let's
just drop the 'architectures' field.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/system/repo/mcp/mirror/v1/openstack.yml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/linux/system/repo/mcp/mirror/v1/openstack.yml b/linux/system/repo/mcp/mirror/v1/openstack.yml
index a4a369b2..933bf53a 100644
--- a/linux/system/repo/mcp/mirror/v1/openstack.yml
+++ b/linux/system/repo/mcp/mirror/v1/openstack.yml
@@ -7,7 +7,6 @@ parameters:
       repo:
         mirantis_openstack:
           source: "deb http://mirror.mirantis.com/${_param:apt_mk_version}/openstack-${_param:openstack_version}/${_param:linux_system_codename} ${_param:linux_system_codename} main"
-          architectures: ${_param:linux_system_architecture}
           clean_file: true
           key_url: https://mirror.mirantis.com/${_param:apt_mk_version}/openstack-${_param:openstack_version}/${_param:linux_system_codename}/archive-${_param:openstack_version}.key
           pin:
